packages/autocomplete-plugin-algolia-insights/src/createAlgoliaInsightsPlugin.ts

```diff
@@ -57,6 +58,10 @@ export type CreateAlgoliaInsightsPluginParams = {
    * @link https://www.algolia.com/doc/ui-libraries/autocomplete/api-reference/autocomplete-plugin-algolia-insights/createAlgoliaInsightsPlugin/#param-insightsclient
    */
   insightsClient?: InsightsClient;
+  /**
+   * Insights parameters to forward to the Insights client’s init method.
+   */
```

Comment:
I'll add a link to the docs

Reply:
can you already do it now? I believe you've made the PR already